tests/test_compute.py

```diff
@@ -78,7 +96,7 @@ def test_compute_norawalgo_allowed(client):
         data=json.dumps(payload),
         content_type='application/json'
     )
-    assert response.status == '400 BAD REQUEST', f'start compute job failed: {response.status} , { response.data}'
+    assert response.status == '400 BAD REQUEST', f'start compute job failed: {response.status} , {response.data}'
```

Comment:
there are also some changes in this file that are not related to the task. Are the changes intended?